### ISSUE TITLE
Stress testing on the 7node example 

### ISSUE THREAD

**Issue by billieluxu** [Consensys-Incorporated/quorum#326]
#### System information

Geth version: `1.5.0-unstable`

OS & Version: OSX

#### Expected behaviour

I was stress testing the throughput of the 7node example by sending 1000 transactions to increase a variable's value by one inside a deployed smart contract. A couple days ago when I tested it it was fine. It took some seconds to finish the for loop and returned a hash to me. Balance was correct after the loop too.

#### Actual behaviour

But starting from yesterday, when I ran the raft-start.sh script, i got the below error when script1.sh tried to send the first transaction:
```
[*] Sending first transaction
err: Error: solc: exit status 1
unrecognised option '--add-std'

false
```
And I got the below error after around sending 200 transactions inside the js console:
```
Error: Non-200 status code: &{Status:500 Internal Server Error StatusCode:500 Proto:HTTP/1.0 ProtoMajor:1 ProtoMinor:0 Header:map[Date:[Wed, 28 Mar 2018 10:11:14 GMT] Server:[Warp/3.2.8] Content-Type:[text/plain; charset=utf-8]] Body:0xc4370633c0 ContentLength:-1 TransferEncoding:[] Close:true Uncompressed:false Trailer:map[] Request:0xc43a3b9c20 TLS:<nil>}
    at web3.js:3122:20
    at web3.js:6026:15
    at web3.js:4998:36
    at web3.js:4058:16
    at apply (<native code>)
    at web3.js:4144:16
    at <anonymous>:1:1
```
After this, I can't even send one transaction anymore.

#### Steps to reproduce the behaviour

To reproduce, please just start a brand new vagrant file and follow the normal steps. But send 1000 transactions instead of 1.

#### Backtrace

````
[backtrace]
````

**Comment by jpmsam** [comment]
@billieluxu I see that you are still running an old version of Quorum, can you please upgrade to the latest master head and run your tests again ?

**Comment by billieluxu** [comment]
@jpmsam thanks! should i just delete the old quorum-example folder and clone the repo again? i dont think there's a upgrade option ...

**Comment by billieluxu** [comment]
I downloaded the latest version and when trying to bring up vagrant, this is the error I got:

```
Vagrant cannot forward the specified ports on this VM, since they
would collide with some other application that is already listening
on these ports. The forwarded port to 22000 is already in use
on the host machine.

To fix this, modify your current project's Vagrantfile to use another
port. Example, where '1234' would be replaced by a unique host port:

  config.vm.network :forwarded_port, guest: 22000, host: 1234

Sometimes, Vagrant will attempt to auto-correct this for you. In this
case, Vagrant was unable to. This is usually because the guest machine
is in a state which doesn't allow modifying port forwarding. You could
try 'vagrant reload' (equivalent of running a halt followed by an up)
so vagrant can attempt to auto-correct this upon booting. Be warned
that any unsaved work might be lost.

```

I assume the vagrant file comes with the quorum package is using port 22000? Previously I managed to bring up the VM and the structure inside the VM is completely different from previous versions - there's no 7nodes example anymore. It could've been there are some installation errors. But in any case, it'd be great if anyone could clarify what has caused the issue.

Thank you,
Billie

**Comment by jpmsam** [comment]
@billieluxu did you destroy or halt the original vagrant box? It seems to be still running and the port forward on the new instance failed.

**Comment by billieluxu** [comment]
@jpmsam Hi I have destroyed and downloaded the new one for the second time and this time it's working. I don't quite understand what happened last time. But I ran into the below problems when tried to run `./runscript.sh script1.js`:

`err creating contract Error: Non-200 status code: &{Status:500 Internal Server Error StatusCode:500 Proto:HTTP/1.1 ProtoMajor:1 ProtoMinor:1 Header:map[Date:[Wed, 18 Apr 2018 08:53:59 GMT] Server:[Warp/3.2.13]] Body:0xc425b75240 ContentLength:-1 TransferEncoding:[chunked] Close:false Uncompressed:false Trailer:map[] Request:0xc437dc7b00 TLS:<nil>}`

Also, the 7node example tutorial is out of date for the new vagrant files. Someone needs to update it, it can be a little misleading now.
